cmd/protoc-gen-gorums/gengorums/template_datatypes.go

```diff
@@ -1,9 +1,30 @@
 package gengorums
 
+import (
+	"github.com/relab/gorums"
+	"google.golang.org/protobuf/compiler/protogen"
+)
+
 var globals = `
 const hasOrderingMethods = {{hasOrderingMethods .Services}}
 `
 
+var orderingInit = `
+{{- range .Services }}
+{{- range orderedMethods .Methods }}
+var {{.GoName}}MethodID int
```

Comment:
I think we can easily make these into `const` instead of variables. Then we don't need the `init()` function, since we can initialize them at generation time.

Reply:
How would you do that? Using a function in `func-map`?

Reply:
I guess I could do something like this:

 `{{- $index, $element := range orderedMethods .Method }}`

 and use `$index` as the ID.

Reply:
Yes, that looks like a reasonable way to do it. Also, do these consts need to be exported?

Reply:
Ah, ok. I see they are just copying the method names and appending MethodID.

Reply:
Perhaps we should use the unexport func on these, to avoid making them exported (exposing them as API)...

Reply:
No, I don't think so. 

By the way, does the `{{- range .Services }}` ever run more than once?

Reply:
No not now, since Gorums currently doesn't support more than one service per proto file. But that's a restriction that could go away in a future upgrade... So, I favor keeping the iterations over all services everywhere we can, so that upgrading it becomes easier in the future...